src/libgraphviz/lib.rs

```diff
@@ -822,10 +833,12 @@ r#"digraph single_cyclic_node {
     #[test]
     fn hasse_diagram() {
         let labels = AllNodesLabelled(vec!("{x,y}", "{x}", "{y}", "{}"));
-        let r = test_input(LabelledGraph::new(
-            "hasse_diagram", labels,
-            vec!(edge(0, 1, ""), edge(0, 2, ""),
-                 edge(1, 3, ""), edge(2, 3, ""))));
+        let r = test_input(LabelledGraph::new("hasse_diagram",
+                                              labels,
+                                              vec!(edge(0, 1, ""),
+                                                   edge(0, 2, ""),
+                                                   edge(1, 3, ""),
+                                                   edge(2, 3, ""))));
```

Comment:
Hmm.  I would have maybe hoped that we would support this:

```rust
+        let r = test_input(LabelledGraph::new(
+            "hasse_diagram",
+            labels,
+            vec!(edge(0, 1, ""), edge(0, 2, ""), edge(1, 3, ""), edge(2, 3, ""))));
```

That is, I think the idiom of putting a line break before the first argument to avoid the heavy indentation of the argument list is a pretty common idiom for us, no?

But maybe it is too hard for rustfmt to decide whether to adopt that strategy or not, since it is often subjective whether to employ it or not.